ros2cli/ros2cli/node/strategy.py

```diff
@@ -43,3 +45,7 @@ def __exit__(self, exc_type, exc_value, traceback):
 def add_arguments(parser):
     add_daemon_node_arguments(parser)
     add_direct_node_arguments(parser)
+    parser.add_argument(
+        '--no-daemon', action='store_true', default=False,
```

Comment:
Nitpick: `default=False` is not necessary. The default `default` value is `None` which would be fine.

Reply:
True, coercion will do the trick. See https://github.com/ros2/ros2cli/pull/420/commits/040646185ccf7c914875658f8c66a55e53982b88.